kernel/data-pipeline/cdc/client/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/client/handler/SubscriptionRequestHandler.java

```diff
@@ -47,8 +47,8 @@ public void userEventTriggered(final ChannelHandlerContext ctx, final Object evt
     
     private CreateSubscriptionRequest buildCreateSubscriptionRequest() {
```

Comment:
Could we remove `buildCreateSubscriptionRequest`? It should be defined by caller

Reply:
Use parameter instead of